app/helpers/ember_rails_helper.rb

```diff
@@ -1,4 +1,24 @@
 module EmberRailsHelper
+  def include_ember_index_html(name)
```

Comment:
Assignment Branch Condition size for include_ember_index_html is too high. [16.12/15]